features_module_how_to_share_sites_config_with_oth.md

```diff
@@ -0,0 +1,130 @@
+# Features module: how to share a site’s configuration with others?
```

Comment:
I think it's worth mentioning this as a D7 solution. Also, would be great to have a session about D8's config :)